src/prefect/exceptions.py

```diff
@@ -276,6 +276,21 @@ class Pause(PrefectSignal):
     """
 
 
+class ExternalSignal(BaseException):
```

Comment:
should these be `PrefectSignals`

Reply:
Maybe? I'm not sure. I think the intent here is to separate this from `PrefectSignal` since we are not the cause of  the signal like for `Abort`.

Reply:
Yep, that's the intent, just to separate external cause from internal cause.